frontend/src/app/components/Header.js

```diff
@@ -74,12 +74,12 @@ export default class Header extends React.Component {
     return (
       <div>
         {this.renderRetireDialog()}
-        <header id="main-header" className="responsive-content-wrapper">
+        <LayoutWrapper id="main-header" flexModifier="row-between-top">
```

Comment:
What about keeping the `header` el and just putting the `LayoutWrapper` inside it? Would be consistent with how `Footer` behaves, and it's nicer to keep the semantically-meaningful `header` vs replacing with a generic `div id="main-header"`.